handlers/interactions/buttonInteractions.ts

```diff
@@ -400,7 +403,10 @@ async function interactionButtonHandler(interaction: ButtonInteraction) {
 function removeIndex(arri: Array<string>, elem: string) {
   let array: Array<string> = [];
 
-  for (let i = 0; i < arri.length && arri[i] != elem; i += 1) {
+  for (let i = 0; i < arri.length; i += 1) {
```

Comment:
```suggestion
  for (let i = 0; i < arri.length; i += 1) {
  if(arri[i] !== elem){
  array.push(arri[i];
```

Reply:
They are both same almost the same login, I would accept your suggestion